### PR TITLE
JEK-47: footer fixed to reflect correct date and company name

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

File: gh-pages-stub/public/js/analytics.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2014-2017 CyberVision, Inc.
+ * Copyright 2016-2018 KaaIoT Technologies, LLC.
```
Comment (rasendubi) [inline gh-pages-stub/public/js/analytics.js:2]: I believe we should specify both companies in there. Do not change the previous line but add a new one with KaaIoT

Comment (hunch7) [reply gh-pages-stub/public/js/analytics.js:2]: Done.

**Comment by Elena363** [comment]
@hunch7 can you, please, merge both commits into one. JEK-47: fixed upon dev rev - is a bad name & I believe this fix doesn't deserve a separate commit )

**Comment by hunch7** [comment]
@Elena363, ok, I squashed the commits into one with a relevant name.

**Pull request merged**